data/navitha2255.jsomn

```diff
@@ -0,0 +1,15 @@
+{
+"name":"navitha kotakonda (text)",
+  "bio":"iam a frontend developer",
+  "links":[ 
+    { "name":"follow me on twitter",
+      "url":https://twitter.com/eddiejaoude",
```

Comment:
Please replace this with your own twitter account. 

You are also missing a `"` in front of `https `